### ISSUE TITLE
ProgressIndicator: Customizing colors doesn't work on IE

### ISSUE THREAD

**Issue by EAlexRojas** [ibm-js/deliteful#566]
Changing the properties related to the text color and lines color as described on documentation:

``` css
.d-progress-indicator-lines {
  stroke: red;
}
.d-progress-indicator text {
  fill: red;
}
```

But on IE, the text and lines are still displayed in black. 

Code used: 

``` html
<d-progress-indicator id="pi" active="true" value="30"></d-progress-indicator>
```

**Comment by EAlexRojas** [comment]
Apparently, on attachedCallback method on the portion of code to work with high contrast windows:

```javascript
this.style.color = window.getComputedStyle(this.msgNode).getPropertyValue("fill");
var currentColor = window.getComputedStyle(this).getPropertyValue("color");
if (this.style.color !== currentColor) {
	this.linesNode.style.stroke = currentColor; // text value color
	this.msgNode.style.fill = currentColor; // lines color
	//android chrome 31.0.1650.59 hack: force to refresh text color otherwise color doesn't change.
	this.msgNode.textContent = this.msgNode.textContent;
}
```

The value gotten on IE for ```this.style.color``` is ```black``` and for ```currentColor``` is ```rgb(0,0,0)```. With that, the style of the linesNode and msgNode related to the color are setted to ```rgb(0,0,0)``` and  the CSS added by the user doesn't have any effect.

**Comment by wkeese** [comment]
Hmm, is this another one of those cases where delite is calling getComputedStyle() and it's not working because the node is hidden (i.e. display:none)?

**Comment by EAlexRojas** [comment]
No, I don't think so. Even if the Element is visible from the beginning, the problem still happens.

**Comment by wkeese** [comment]
OK... and presumably you are talking about this happening when the computer *isn't* in high-contrast mode?

I see how the code above is trying to detect the high-contrast text color, and to use it for the ProgressIndicator.  It's an interesting idea, but tricky, so I'm not surprised that it's not working. ISTM there are two issues:

- The first problem is that the code you quoted mistakenly detects that the display is in high-contrast mode whereas really it isn't.  I assume the high-contrast detection code in dojo works fine.

- When the display is actually in high-contrast mode, the code above apparently doesn't detect the high-contrast mode text color correctly.   Not sure if that's possible to fix.   An alternate approach would be to hard-code both foreground and background colors in high-contrast mode, making the text black and background white (or vice-versa).  The important thing would be to hardcode both foreground and background, rather than having a transparent background, to ensure proper contrast.

**Comment by wkeese** [comment]
This was fixed by f65c958741944fa4611d8734cd1b366555f808f4.
